docs/SplunkOperatorUpgrade.md

```diff
@@ -0,0 +1,77 @@
+# How to upgrade Splunk Operator and Splunk Enterprise Deployments
+​
+To upgrade the Splunk Operator for Kubernetes, you will overwrite the prior operator release with the latest version. Once the latest version of the Splunk Operator is running, the CRD's are updated and any running pods in the operator managed namespace are restarted. Any new spec defined by the operator will be applied to the restarted pods.
+​
+A Splunk Operator for Kubernetes upgrade might include support for a later version of the Splunk Enterprise Docker image. After the Splunk Operator completes its upgrade, the pods will be restarted using the latest Splunk Enterprise Docker image.
+​
+* Note: The Splunk Operator does not provide a way to downgrade to a previous release.
+​
+## Prequisites
+​
+* Before you upgrade, review the Splunk Operator [change log](https://github.com/splunk/splunk-operator/releases) page for information on changes made in the latest release. The Splunk Enterprise Docker image compatibility is noted in each release version.
+​
+* If the Splunk Enterprise Docker image changes, review the Splunk Enterprise [Upgrade Readme](https://docs.splunk.com/Documentation/Splunk/latest/Installation/AboutupgradingREADTHISFIRST) page before upgrading. 
+​
+* For general information about Splunk Enterprise compatibility and the upgrade process, see [How to upgrade Splunk Enterprise](https://docs.splunk.com/Documentation/Splunk/latest/Installation/HowtoupgradeSplunk).
+​
+* If you use forwarders, verify the Splunk Enterprise version compatibility with the forwarders in the [Compatibility between forwarders and Splunk Enterprise indexers](https://docs.splunk.com/Documentation/Forwarder/latest/Forwarder/Compatibilitybetweenforwardersandindexers) documentation.
+​
+## Upgrading Splunk Operator and Splunk Operator Deployment
+​
+1. Download the latest Splunk Operator installation yaml file.
+​
+```
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/0.2.2/splunk-operator-install.yaml 
```

Comment:
Nit: **`wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/<desired_version_of_operator>/splunk-operator-install.yaml `**?

Reply:
That will be a non-working link. I am under the impression we will update this URL with new release. @smohan-splunk Can you please confirm how we want to approach this ?

Reply:
Leaving at 0.2.2.